samples/net/openthread/coap_client/src/coap_client.c

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(coap_client, LOG_LEVEL_DBG);
+
+#include <logging/log_ctrl.h>
+#include <errno.h>
+#include <zephyr.h>
+#include <net/socket.h>
+
+#include "board_utils.h"
+#include "coap_utils.h"
+
+#define SLEEP_TIME 500
+#define IPV6_ADDR_LENGHT 16
+
+/**@brief Enumeration describing light commands. */
+typedef enum {
+	THREAD_COAP_UTILS_LIGHT_CMD_OFF = '0',
```

Comment:
IMO: This is an external api so all values shall be specified, values should not shift when something is added in the middle. In theory server may be using other enum version and adding new values shall not brake old functionality.

Consider also moving this to separate file as a server will also require this enum.

Reply:
done